### PR TITLE
Simulator Radio General Maintenance

### PR DESCRIPTION
**Pull request by alexhorner**
- New images for the store
- Added some more tags
- Set the metadata schema
- Updated to support something upcoming
- Some general cleanup bits

### CODE REVIEW

**Comment by alexhorner** [comment]
premid1
premid2
premid3

**Comment by alexhorner** [comment]
Also updated the application for our new logo

premid

File: websites/S/Simulator Radio/dist/metadata.json [hunk 1]
```diff
@@ -14,9 +15,9 @@
     "en": "Simulator Radio | Your #1 Simulation Station"
   },
   "url": "simulatorradio.com",
-  "version": "1.4.4",
-  "logo": "https://i.imgur.com/5wpjeAM.png",
-  "thumbnail": "https://i.imgur.com/7OIweId.png",
+  "version": "1.5.0",
+  "logo": "https://i.imgur.com/xjgquwP.png",
+  "thumbnail": "https://i.imgur.com/QIy2Kfe.png",
```
Comment (Bas950) [inline websites/S/Simulator Radio/dist/metadata.json:20]: Nice logo 
image

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/S/Simulator Radio/dist/metadata.json (hunk 1 above)
Comment (alexhorner) [reply websites/S/Simulator Radio/dist/metadata.json]: Hi Bas, sorry about that, not sure why that link went invalid but I've set a new one now

**Review by Bas950 — APPROVED** [review]
LGTM

**Review by EncryptedDev — DISMISSED** [review]

File: websites/S/Simulator Radio/dist/metadata.json [hunk 2]
```diff
@@ -14,9 +15,9 @@
     "en": "Simulator Radio | Your #1 Simulation Station"
   },
   "url": "simulatorradio.com",
-  "version": "1.4.4",
-  "logo": "https://i.imgur.com/5wpjeAM.png",
-  "thumbnail": "https://i.imgur.com/7OIweId.png",
+  "version": "1.5.0",
+  "logo": "https://i.imgur.com/xjgquwP.png",
```
[the remark this answers is not in the document]
Comment (Bas950) [reply websites/S/Simulator Radio/dist/metadata.json]: lol it was working a few mins ago...

[the remark this answers is not in the document]
Comment (alexhorner) [reply websites/S/Simulator Radio/dist/metadata.json]: For goodness sake. Do you guys recommend any other upload service?

[the remark this answers is not in the document]
Comment (ririxidev) [reply websites/S/Simulator Radio/dist/metadata.json]: Give this logo here, I will upload it.

[the remark this answers is not in the document]
Comment (alexhorner) [reply websites/S/Simulator Radio/dist/metadata.json]: @ririxidev I feel like some kind of copyright protection is taking the images down which is totally possible.

SRLogo

[the remark this answers is not in the document]
Comment (ririxidev) [reply websites/S/Simulator Radio/dist/metadata.json]: https://i.imgur.com/fUvjEoV.png if something, literally try uploading different logo :/

[the remark this answers is not in the document]
Comment (ririxidev) [reply websites/S/Simulator Radio/dist/metadata.json]: let's wait an hour and see if it gets deleted too.

**Review by ririxidev — APPROVED** [review]
lgtm

**Review by StrikerFRFX — APPROVED** [review]
lgtm

**Pull request merged**